### PR TITLE
release context changes

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: This is a branch to group breaking changes

**Why**: So we can limit how many major releases we push

**How**: Merged a breaking change into `next`.

What else could we do as part of this major release? I'll wait about 24 hours before merging/releasing this.

### CODE REVIEW

**Comment by kentcdodds** [comment]
@ajwhite, thanks so much for your all your help! I've added you as a collaborator on the project. Please make sure that you review the MAINTAINING.md file (specifically the bit about the commit messages and the git hooks) and familiarize yourself with the code of conduct (we're using [the contributor covenant](http://contributor-covenant.org/)). You might also want to [watch the repo](https://help.github.com/articles/watching-repositories/) to be notified when someone files an issue/PR. Please continue to make PRs as you feel the need (you can make your branches directly on the repo rather than your fork if you want). Thanks! And welcome to the team :)

**Comment by kentcdodds** [comment]
Oh, and for this PR I'm going to go ahead and merge it myself, but I would like a review :)

File: README.md [hunk 1]
```diff
@@ -682,6 +682,40 @@ class SubTitle extends Component {
...
 
+### Context
+
+[context](https://facebook.github.io/react/docs/context.html) is an unstable
+API and it's not recommended to use it directly. However, if you need to use it
+for some reason, here's an example of how you could do that:
+
+```jsx
+const dynamicStyles = (props, theme, context) => (
+  <div>{context.foo}</div>
```
Comment (ajwhite) [inline README.md]: The _dynamic style function_ can return an HTML element, rather than a style?

Comment (kentcdodds) [inline README.md]: lol, whoops!

File: README.md [hunk 2]
```diff
@@ -682,6 +682,40 @@ class SubTitle extends Component {
...
+    return <MyDiv />
+  }
+}
+
+Parent.childContextTypes = {
+  isLoggedIn: PropTypes.string,
+}
+
+<Parent />
+// renders <div>Hello world</div>
```
Comment (ajwhite) [inline README.md]: With the last change - may want to update this for clarity?

```
// renders <div />
// with {color: 'green'}
```

**Comment by ajwhite** [comment]
RE https://github.com/paypal/glamorous/pull/128#issuecomment-303751668

Sounds great! Already watching, and will familiarize with the documents mentioned 💯

**Comment by ajwhite** [comment]
RE https://github.com/paypal/glamorous/blob/master/other/MAINTAINING.md#release

Do caution that 5a0a9f3ce0bda4456a18729b1a8afa9039114831 mentions `BREAKING CHANGE`

**Comment by ajwhite** [comment]
I assume this will be cleaned up in the squash & merge commit?

**Comment by kentcdodds** [comment]
Yep! That's why I said I wanted to merge this one myself :) So I could make sure that happens. In fact, glamorous has _never_ made a breaking change, even though we're on v3. The two major version bumps so far were a mistake on my part of merging PRs that included `BREAKING CHANGE` 😅

**Pull request merged**